apidiscovery/templates/processor-configmap.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: processor-configmap
+  namespace: {{ .Values.discovery.namespace }}
+data:
+  processor-cm.yaml: |
+{{- if (eq .Values.discovery.processor "enabled") }}
+    processors:
+      batch:
+        send_batch_size: 10000
+        send_batch_max_size: 11000
+        timeout: "30s"
+      memory_limiter:
+        check_interval: "1s"
+        limit_mib: 256
```

Comment:
Add a doc note to suggest that the limit_mib can be increased if the customer ever updates the resource limits of the collector